tests/bsim/bluetooth/host/l2cap/send_on_connect/CMakeLists.txt

```
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(bsim_test_l2cap_send_on_connect)
+
+FILE(GLOB app_sources src/*.c)
```

```suggestion
FILE(GLOB app_sources src/*.c)
target_sources(app PRIVATE ${app_sources})

target_sources(app PRIVATE
    main.c
    common.c
    main_l2cap_send_on_connect.c
)
```
We try to avoid usage of globbing.

---

No problem, fixed :)